dwebsummit/dwebsummit_frontend/templates/dwebsummit/_sponsorship_section.html

```diff
@@ -0,0 +1,12 @@
+<section class="{{section_class}}">
+  <h1>{{section_title}}</h1>
+  <div class="sponsors-grid">
+    {% for sponsor in sponsors %}
+      <div id="sponsor_{{sponsor.slug}}" class="box">
```

Comment:
Suggestion: Another copy paste, I know, but if easy, consider replacing “id” with “class”.

Reply:
It doesn't look like we were using these IDs so I removed them